kibbeh/public/locales/id/translation.json

```
@@ -256,7 +273,7 @@
         "debugAudio": "Debug Audio",
         "stopDebugger": "Berhentikan Debugger"
       },
-      "downloadApp": "Download App"
+      "downloadApp": "Undah aplikasi"
```

```suggestion
      "downloadApp": "Unduh aplikasi"
```
is this a typo? shouldn't this be "Unduh"